Implementacija/savetovaliste/app/Views/odgovori_na_pitanje.php

```diff
@@ -0,0 +1,333 @@
+
+        <div class="w3-left" style="padding-left: 5%;"> 
+            <h3 class="letters_dark_blue"> <b> ' . $pitanje->naslovPitanja . '   </b>  </h3>
+        </div>
+    ';   
+    
+    // ako je anonimno postavljeno pitanje napisi anonimno
+    if($pitanje->postavljenoAnonimno==1)
+    {
+        echo '
+            <div class="w3-right" style="padding-right: 6%;"> 
+                <h3 class="letters_dark_blue"><b> ' . "Anonimno" . '</b></h3>
+            </div> ';
+    }
+    else
+    {
+        // ako nije anonimno stavi autora pitanja
+        $korisnikModel=new KorisnikModel();
+        $idAutora=$pitanje->korisnik_idKorisnik_postavio;
+        $autor=$korisnikModel->findUserUsername($idAutora);
+        echo '
+            <div class="w3-right" style="padding-right: 6%;"> 
+                <h3 class="letters_dark_blue"><b>' . $autor . '</b></h3>
+            </div>
+        ';
+    }
+
+    // da li svi ili samo registrovani korisnici mogu da odgovore na pitanje
+    $sviImajuPravoDaOdgovore="Svi registrovani korisnici";
+    if ($pitanje->moguSviDaOdgovore==0) $sviImajuPravoDaOdgovore="Samo registrovani psiholozi";
+
+    echo '
+    <div class="input letters_dark_blue">
+    <p style="text-align: justify; font-weight: normal;"> <b> Na ovo pitanje mogu da odgovore: ' . $sviImajuPravoDaOdgovore . '</></p> 
+    </div>';
+   
+    $referencaOdgovori=site_url("$controller/pregledOdgovora?pretraga=$idPitanja");
+    // prikaz teksta pitanja
+    echo '
+        <div class="input letters_dark_blue">
+            <p style="text-align: justify; font-weight: normal;"> ' . $pitanje->tekstPitanja . ' </p> 
+        </div>
+        ';
+
+    // prikaz broja lajkova i dislajkova 
+    echo '
+    <br>
+    <div class="input">     
+        <div id="like">
+            <div>
+                <!-- TODO ubaciti lajkovanje za ulogovanog korisnika -->
+                <button class="w3-button buttons" onclick=""><b><i class="fa fa-thumbs-up"></i> <u onclick="" style="text-decoration: none; font-weight: normal;">Korisno (' . "$likes" . ')</u></button> &nbsp
+                <!-- TODO ubaciti lajkovanje za ulogovanog korisnika -->
+                <button class="w3-button buttons" onclick=""><b><i class="fa fa-thumbs-down"></i> <u onclick="" style="text-decoration: none; font-weight: normal;">Nije korisno (' . "$dislikes" . ')</u></button>
+            </div>
+    
+            <div style="float: right;">                                    
+            <button onclick="" class="w3-button buttons" style="font-weight: normal;"> 
+                <a class="nema_podvlacenja" href=' . "$referencaOdgovori" . '>Pogledaj odgovore</a>
+            </button> &nbsp
+           
+        </div> 
+        </div>
+    <br>
+    </div>
+
+    </div> <!-- Kraj div container -->
+    ';
+    
+?>
+       
+  
+    <?php  $idPitanja=$pitanje->idPitanje;  ?> 
+    <form name="odgovoriNaPitanje"  action="<?= site_url("$controller/odgovoriNaPitanje/$idPitanja") ?>">
```

Comment:
Opet method post, da ne bi išlo kroz URL.
```suggestion
    <form name="odgovoriNaPitanje"  action="<?= site_url("$controller/odgovoriNaPitanje/$idPitanja") ?>" method="post">
```

Reply:
Sredjeno

Reply:
Sredjeno!